manifests/init.pp

```diff
@@ -90,6 +90,9 @@
 # $ca_key_password::              CA key password
 #                                 type:String
 #
+# $ciphers::                      Allowed ciphers for ssl connection. Array or strings
```

Comment:
Did you mean 'of strings'? If so you can make the type ```type:Array[String]``` for additional checking.